wordpress-mobile/WordPress-Android
# Update icons set

This changes the app icons set to cool blue.

**NOTE: a couple of things must be double checked before merge:**
- ~~the assets for API Nougat and below must be double checked and are temporary! They miss transparency and some fine adjustment on pixel dimension~~ updated the resources with correct one
- ~~the fill color for foreground app icon~~ color is now correctly set to blue_50

| Scenario | | | |
|---|---|---|---|
| Oreo or above | ![image](https://user-images.githubusercontent.com/47797566/150440455-1d4b8b99-fedf-4f9f-8be8-92b5a3c287f2.png) | ![image](https://user-images.githubusercontent.com/47797566/150440147-fd602f2a-681b-4d5d-adee-e52a5043932f.png) | ![image](https://user-images.githubusercontent.com/47797566/150440199-810cf541-eb17-4b27-b80c-282ad9031ffc.png) |
| Nougat or below | ![image](https://user-images.githubusercontent.com/47797566/150440530-bc7a720b-12fc-46db-9a3e-0f52738a9f3c.png) | ![image](https://user-images.githubusercontent.com/47797566/150440749-ad20978c-e2a0-4b68-a464-5ab27e286075.png) | ![image](https://user-images.githubusercontent.com/47797566/150441504-5a239c18-3e9e-4e19-b987-94fd9e11c60a.png) |


### To test:
- Install the apk on low and high resolution simulators (both with Nougat or below API and Oreo or above API).
- Check the icon is the new one as expected.

## Regression Notes
1. Potential unintended areas of impact
None if properly tested on the various resolutions and API levels threshold.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

3. What automated tests I added (or what prevented me from doing so)
N/A


PR submission checklist:
